src/main/java/budgetbuddy/transaction/TransactionList.java

```diff
@@ -171,8 +171,29 @@ public static ArrayList<Transaction> getCustomDateTransactions(ArrayList<Transac
         return customDateTransactions;
     }
 
+    public static ArrayList<Transaction> getAccountTransactions(ArrayList<Transaction> transactions,
+                                                                int accountNumber) {
+        ArrayList<Transaction> accountTransactions = new ArrayList<>();
+        for (Transaction transaction : transactions) {
+            if (transaction.getAccountNumber() == accountNumber) {
+                accountTransactions.add(transaction);
+            }
```

Comment:
Good job on adding this option. It will surely helps the user to go through their transactions more efficiently.